pandas/core/frame.py

```diff
@@ -471,7 +477,7 @@ def __init__(self, data=None, index=None, columns=None, dtype=None,
     # ----------------------------------------------------------------------
 
     @property
-    def axes(self):
+    def axes(self) -> List[ArrayLike]:
```

Comment:
see comment above, though this actually is List[Index]

Reply:
Alright, thanks, I will fix all these as soon as I get some time. Right now too busy finding a new job 😭